### PR TITLE
adopted for official

### PR DESCRIPTION
**Pull request by dhruvkumar1805**

### CODE REVIEW

**Review by gwolf2u — CHANGES_REQUESTED** [review]
Please remove new files from initial folder
Those are not needed

File: ysl.json [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+  "response": [
+    {
+        "maintainer": "Dhruv Chhura (Dhruv)",
+        "oem": "Xiaomi",
+        "device": "Redmi S2/Y2",
+        "filename": "crDroidAndroid-11.0-20210122-ysl-v7.2.zip",
+        "download": "https://sourceforge.net/projects/crdroid/files/ysl/7.x/crDroidAndroid-11.0-20210122-ysl-v7.2.zip/download",
+        "timestamp": 20210122,
+        "md5": "ca4d75993b3fcd86cdfb9ecd823d5ef2",
+        "size": 783 MB,
```
Comment (gwolf2u) [inline ysl.json]: Use generated size

File: ysl.json [hunk 2]
```diff
@@ -0,0 +1,24 @@
+{
+  "response": [
+    {
+        "maintainer": "Dhruv Chhura (Dhruv)",
+        "oem": "Xiaomi",
+        "device": "Redmi S2/Y2",
+        "filename": "crDroidAndroid-11.0-20210122-ysl-v7.2.zip",
+        "download": "https://sourceforge.net/projects/crdroid/files/ysl/7.x/crDroidAndroid-11.0-20210122-ysl-v7.2.zip/download",
+        "timestamp": 20210122,
```
Comment (gwolf2u) [inline ysl.json]: Use generated timestamp

**Review by gwolf2u — CHANGES_REQUESTED** [review]

**Review by dhruvkumar1805 — COMMENTED** [review]
Added generated timestamp and size

**Pull request merged**